tests/test_attributes.py

```diff
@@ -101,7 +101,7 @@ def get_context_data(self, *args, attrs):
         self.assertHTMLEqual(
             rendered,
             """
-            <div @click.stop="dispatch('click_event')" x-data="{hello: 'world'}" class="padding-top-8 added_class another-class" data-id=123>
+            <div @click.stop="dispatch('click_event')" x-data="{hello: 'world'}" class="padding-top-8 added_class another-class" data-djc-id-a1bc3f data-id=123>
```

Comment:
There's A LOT of changes to the tests, because now, when you render a component, it's root elements will be tagged with the component ID.

So if a component has template

```djangohtml
<div>{{ message }}<div>
<span> Sent from <b>iPhone</b> </span>
```

Then the end result will be

```djangohtml
<div data-djc-id-1bc3df> hi! <div>
<span data-djc-id-1bc3df> Sent from <b>iPhone</b> </span>
```